src/psyclone/psyGen.py

```diff
@@ -2464,7 +2461,23 @@ def __init__(self, call, arg_info, access):
         self._precision = None
         self._data_type = None
         self._module_name = None
+        # Default the name to the original name for debugging
+        # purposes. This may be updated when arg_init1() is called.
+        self._name = self._orig_name
+
+    def arg_init1(self, arg_info):
```

Comment:
Please rename this (maybe "init_arg_symbol"?) Could it also be private?

Reply:
As mentioned earlier I've renamed it, and pre-pended an underscore.